clustering.py

```diff
@@ -0,0 +1,36 @@
+# import libraries
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.cluster import KMeans
+from scipy import stats
+import seaborn as sns
+
+
+def make_map():
+    data = pd.read_csv('example/out.tips')
```

Comment:
Perhaps it would be useful here to make this more flexible -- so that we can cluster any `.tips` file?
We can do this with an argument parser like [`argparse`](https://docs.python.org/3/library/argparse.html) to do this pretty quickly.

Reply:
I'll get started on this. Great suggestion!